### PR TITLE
Fix issue #88 - Release set with milestone rather than target_release on BZ issue

### PR DESCRIPTION
**Pull request by rpelisse**

### CODE REVIEW

**Comment by baranowb** [comment]
@rpelisse :this needs some care.

**Comment by rpelisse** [comment]
noted - I'll rework this next week.

**Comment by rpelisse** [comment]
@baranowb OK, should look better now :)

File: src/test/java/org/jboss/set/aphrodite/issue/trackers/jira/JiraIssueWrapperTest.java [hunk 1]
```diff
@@ -222,7 +222,7 @@ private Issue createTestIssue01(URL url) throws ParseException {
         result.setType(org.jboss.set.aphrodite.domain.IssueType.BUG);
 
         List<Release> releases = new ArrayList<>();
-        releases.add(new Release("6.4.4"));
+        releases.add(new Release("6.4.5"));
```
Comment (baranowb) [inline src/test/java/org/jboss/set/aphrodite/issue/trackers/jira/JiraIssueWrapperTest.java]: why?

Comment (baranowb) [inline src/test/java/org/jboss/set/aphrodite/issue/trackers/jira/JiraIssueWrapperTest.java]: Merged already, I was just curious.

File: src/main/java/org/jboss/set/aphrodite/issue/trackers/bugzilla/IssueWrapper.java [hunk 2]
```diff
@@ -110,9 +110,10 @@ Issue bugzillaBugToIssue(Map<String, Object> bug, URL baseURL) {
         String type = (String) bug.get(ISSUE_TYPE);
         issue.setType(IssueType.getMatchingIssueType(type));
 
-        setAffectedVersions(issue, (Object[]) (bug.get(VERSION)) );
-        setReleases(issue, bug);
-
+        String version = (String) ((Object[]) bug.get(VERSION))[0];
```
Comment (soul2zimate) [inline src/main/java/org/jboss/set/aphrodite/issue/trackers/bugzilla/IssueWrapper.java]: version is never used here.

File: src/main/java/org/jboss/set/aphrodite/issue/trackers/bugzilla/IssueWrapper.java [hunk 3]
```diff
@@ -110,9 +110,10 @@ Issue bugzillaBugToIssue(Map<String, Object> bug, URL baseURL) {
...
+        List<Release> releases = new ArrayList<>();
+        releases.add(new Release((String) bug.get(TARGET_RELEASE), (String) bug.get(TARGET_MILESTONE)));
```
Comment (rpelisse) [inline src/main/java/org/jboss/set/aphrodite/issue/trackers/bugzilla/IssueWrapper.java]: @soul2zimate Thanks for spotting this ! I spent my day yesterday trying to make sense of this, and I found out that this all PR was in fact superseed a while back. My attempt to fix it just made things worse. I'm going to revert it - there is not a single line of code useful in it.

**Pull request merged**